### ISSUE TITLE
low validation accuracy of gcn compared to random forest

### ISSUE THREAD

**Issue by Leungfranklin** [rusty1s/pytorch_geometric#1367]
## ❓ Questions & Help

I have tried gcn in a node classification scenario. To my surprise, the performance accuracy ( both the training and the validation dataset ) is lower than the accuracy I get just using random forest on my node attributes data. Is that normal? theoretically the graph information should help the classification but I do not observe such benefits in applying gcn.

**Comment by rusty1s** [comment]
That is actually really hard to tell. In general, GNN help dramatically to achieve better results if your input has useful relational information. Your performance may be explained by wrong hyper parameters, overfitting or a wrong GNN operator. For example, `GCNConv` is not good at preserving central node information, so you may want to test a different one, *e.g.*, `SAGEConv`.

**Comment by Leungfranklin** [comment]
Thanks, have much more to test
